ceremonies/src/lib.rs

```diff
@@ -125,14 +125,16 @@ decl_storage! {
         ParticipantIndex get(fn participant_index): double_map hasher(blake2_128_concat) CurrencyCeremony, hasher(blake2_128_concat) T::AccountId => ParticipantIndexType;
         ParticipantCount get(fn participant_count): map hasher(blake2_128_concat) CurrencyCeremony => ParticipantIndexType;
         ParticipantReputation get(fn participant_reputation): double_map hasher(blake2_128_concat) CurrencyCeremony, hasher(blake2_128_concat) T::AccountId => Reputation;
+        // newbies granted a ticket from a bootstrapper for the next ceremony.
+        Endorsees get(fn endorsees): map hasher(blake2_128_concat) CurrencyCeremony => Vec<T::AccountId>;
```

Comment:
have you considered solving this with a new variant in the Reputation enum? Vec is ok as the size is limited, but actually, an endorsement is a form of reputation, no?

Reply:
Ah nice idea, I must admit I did not think of that. I can't think of a reason to not use an enum variant.

Reply:
Then we should think about how we want to handle now-shows of endorsees in the future. If we don't want to forfeit the endorsement, how do we know the ceremony_index at which he has been endorsed?

I guess, the registering participant would need to supply a 'Proof_of_Endorsement'?

Reply:
that ProofOfEndorsement would be the consequence of using an enum variant, yes. Your `vec` approach is more straightforward I must admit.

Let's consider the act of endorsing:

Your `vec` version means the endorsee shows a QR code to the endorser who then submits a tx. The endorsee registers like everyone else.

ProofOfEndorsement is slightly more involved: The endorsee has to know he's been endorsed and register differently.

I think this just adds complexity and doesn't help with sclability, as it is only used in the initial phase. Let's stick with your `vec` version

Reply:
Yes, I would opt for the simpler version too.